Allow users to show or hide havbar icons (#39)
pyansys/pyansys-sphinx-theme: doc/source/usage.rst

If you want to use the version number in
the ``index.rst`` title, use ``|version|`` to include the package version
number.

Custom Icons
------------

> This theme allows you to have a huge control over the icons displayed in the
> navigation bar.

Reviewer: ```suggestion
This theme allows you to have an accurate control over the icons displayed in the
navigation bar.
```
